packages/lib/src/dev/expose-development.ts

```diff
@@ -52,11 +52,11 @@ export function devExposePlugin(
       };
       const __federation_import = async (urlImportPath, fsImportPath) => {
         let importedModule;
-        try {
-          return await import(fsImportPath);
-        }catch(ex) {
-          return await import(urlImportPath);
-        }
+          try {
```

Comment:
Does this indentation have any meaning?

If you'd like to consider having this commit considered as a contribution to the main fork, best to only change what you must.